scene/2d/cpu_particles_2d.h

```diff
@@ -73,9 +73,16 @@ class CPUParticles2D : public Node2D {
 		EMISSION_SHAPE_RECTANGLE,
 		EMISSION_SHAPE_POINTS,
 		EMISSION_SHAPE_DIRECTED_POINTS,
+		EMISSION_SHAPE_CIRCLE,
 		EMISSION_SHAPE_MAX
 	};
 
+	enum EmissionCircleMode {
+		EMISSION_CIRCLE_MODE_RANDOM,
+		EMISSION_CIRCLE_MODE_CW,
+		EMISSION_CIRCLE_MODE_CCW
```

Comment:
```suggestion
		EMISSION_CIRCLE_MODE_CCW,
```
You could leave a trailing comma in case more modes are added.